locust/contrib/mongo.py

```diff
@@ -0,0 +1,47 @@
+from gevent import monkey
+
+_ = monkey.patch_all()
+
+from locust import User, events
+
+import time
+
+from pymongo import MongoClient
+from pymongo.errors import PyMongoError
+
+
+class MongoDBClient:
+    def __init__(self, conn_string, db_name):
+        self.client = MongoClient(conn_string)
```

Comment:
It is more similar to HttpUser and others if User.client *IS* a MongoClient subclass (with overridden method for execute_query calling the superclass) rather than *containing* a MongoClient. Check HttpUser for example.